docs/cadence/language/values-and-types.mdx

```diff
@@ -1253,7 +1253,7 @@ It is invalid to use one of these functions on a fixed-sized array.
 
 -
     ```cadence
-    fun append(_ element: T): Void
+    access(Mutate | Insert) fun append(_ element: T): Void
```

Comment:
Could `Mutate | Insert` here and below just be `Insert`, given `Mutate` effectively is/implies `Insert, Remove`? Or is `Mutate` mentioned just for documentation purposes, e.g. to avoid confusion like someone not realizing that fact and thinking "I need `Insert`, but only have `Mutate`"?

(Same for the `Mutate | Remove` cases BTW)

Reply:
> `Mutate` effectively is/implies `Insert, Remove`

Unfortunately it doesn't literally work this way in the language; `Mutate` isn't actually an entitlement that inherits `Insert` and `Remove`, because we don't support that. We just use it as such in all function definitions for containers to mimick the behavior.

Reply:
I see! I had assumed we had special support for this, but totally fine (and maybe even better) if not. Thank you for the clarification 🙏